var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -73,6 +73,11 @@ class Gcc(AutotoolsPackage):
     variant('piclibs',
             default=False,
             description='Build PIC versions of libgfortran.a and libstdc++.a')
+    if sys.platform == 'darwin':
```

Comment:
also please add a comment above that one has to set extra rpaths (just so that it's easier to find later).